app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  validates :name, presence:true
+  validates :address, presence:true
+  validates :city, presence:true
+  validates :state, presence:true
+  validates :zip, presence:true
+  validates :email, presence:true, uniqueness:true
+  validates :password, presence:true
+  validates :password_confirmation, presence:true
+
```

Comment:
I am also curious if this password confirmation is a value that is supposed to be validated... seems redundant but also worth inquiring about...